cmd/install.go

```diff
@@ -205,40 +208,59 @@ func runInstallSuperNodeRemoteSubCommand(ctx context.Context, config *configs.Co
 
 	pastelUtilityPath := filepath.Join(config.RemotePastelUtilityDir, "pastel-utility")
 	pastelUtilityPath = strings.ReplaceAll(pastelUtilityPath, "\\", "/")
-	pastelUtilityDownloadPath := constants.PastelUtilityDownloadURL
 
-	_, err = client.Cmd(fmt.Sprintf("rm -r -f %s", pastelUtilityPath)).Output()
+	err = client.ShellCmd(ctx, fmt.Sprintf("rm -r -f %s", pastelUtilityPath))
 	if err != nil {
 		log.WithContext(ctx).Error("Failed to delete pastel-utility file")
 		return err
 	}
 
-	log.WithContext(ctx).Info("Downloading Pastel-Utility Executable...")
-	_, err = client.Cmd(fmt.Sprintf("wget -O %s %s", pastelUtilityPath, pastelUtilityDownloadPath)).Output()
+	// Download pastel-ultility from pastel website
+	if config.DisableTransferLocal {
+		pastelUtilityDownloadPath := constants.PastelUtilityDownloadURL
+		log.WithContext(ctx).Info("Downloading Pastel-Utility Executable...")
+		err = client.ShellCmd(ctx, fmt.Sprintf("wget -O %s %s", pastelUtilityPath, pastelUtilityDownloadPath))
 
-	log.WithContext(ctx).Debugf("wget -O %s  %s", pastelUtilityPath, pastelUtilityDownloadPath)
-	if err != nil {
-		log.WithContext(ctx).Error("Failed to download pastel-utility")
-		return err
-	}
+		log.WithContext(ctx).Debugf("wget -O %s  %s", pastelUtilityPath, pastelUtilityDownloadPath)
+		if err != nil {
+			log.WithContext(ctx).Error("Failed to download pastel-utility")
+			return err
+		}
+		log.WithContext(ctx).Info("Finished Downloading Pastel-Utility Successfully")
+	} else {
+		// scp pastel-ultility to remote
+		log.WithContext(ctx).Info("Transferering local Pastel-Utility Executable to remote")
+		err = client.Scp(os.Args[0], pastelUtilityPath)
+		if err != nil {
+			log.WithContext(ctx).WithError(err).Error("Failed to Transferering local Pastel Utility to remote")
+			return err
+		}
 
-	log.WithContext(ctx).Info("Finished Downloading Pastel-Utility Successfully")
+		log.WithContext(ctx).Info("Finished Transferering local Pastel-Utility Successfully")
+	}
 
-	_, err = client.Cmd(fmt.Sprintf("chmod 777 /%s", pastelUtilityPath)).Output()
+	err = client.ShellCmd(ctx, fmt.Sprintf("chmod 777 /%s", pastelUtilityPath))
```

Comment:
seem redundant `/`

Reply:
but no impact at all. because this pastelUtilityPath is absolute path. will fix it later in other PR